dist/stat_data.py

```diff
@@ -498,6 +508,28 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_write_restore', 'pages written requiring in-memory restoration'),
 
     ##########################################
+    # Cache content statistics
+    ##########################################
+    CacheWalkStat('cache_state_avg_written_size', 'Average on-disk page image  size seen', 'no_clear,no_scale'),
+    CacheWalkStat('cache_state_count_memory', 'Number of pages that have never been written from cache', 'no_clear,no_scale'),
+    CacheWalkStat('cache_state_count_not_queueable', 'Number of pages that could not be queued for eviction', 'no_clear,no_scale'),
+    CacheWalkStat('cache_state_count_queued', 'Number of pages currently queued for eviction', 'no_clear,no_scale'),
+    CacheWalkStat('cache_state_count_smaller_alloc_size', 'Number of on-disk page image sizes smaller than a single allocation unit', 'no_clear,no_scale'),
```

Comment:
In all of the above, "count" seems redundant in the name.  We mostly assume stats are a count unless they have "size" in them, I think...